pynubank/nubank.py

```diff
@@ -127,6 +130,13 @@ def get_bills(self):
         else:
             raise NuMissingCreditCard
 
+    def get_customer(self):
+        if self.customer_url is not None:
```

Comment:
It's possible for this URL to be `None` ?

Reply:
Yes, i have the same question myself. I don't believe so... Should remove the check them and assume it must be there since customer is a must-have metadata anyway